azure-pipelines.sds.pr-release.yml

```diff
@@ -103,6 +104,7 @@ stages:
 
           - template: templates\Containerisation\docker\docker-compose.yaml@azTemplates
             parameters:
+              dockerComposeFile: $(System.DefaultWorkingDirectory)/docker-compose.yml
```

Comment:
think this can be removed now that it has been added to the devops template branch